site/docs/master/oracle-config.md

```diff
@@ -0,0 +1,243 @@
+# Use Oracle Cloud as a Backup Storage Provider for Velero
+
+## Introduction
+
+[Velero](https://heptio.github.io/velero/master/) is a tool used to backup and migrate Kubernetes applications. Here are the steps to use [Oracle Cloud Object Storage](https://docs.cloud.oracle.com/iaas/Content/Object/Concepts/objectstorageoverview.htm) as a destination for Velero backups. 
```

Comment:
Given that this page will already be on the website, I'm not sure it needs to be a link, but if you want to leave the link, let's set this to just https://velero.io